token/cli/src/main.rs

```diff
@@ -1351,6 +1351,7 @@ fn command_close(
     recipient: Pubkey,
     bulk_signers: BulkSigners,
 ) -> CommandResult {
+    let mut is_recipient_wrapped = false;
```

Comment:
Nit to avoid mutables:
```suggestion
    let is_recipient_wrapped = if config.sign_only {
        false
    } else {
    ... the rest of the block as normal ...
    };
```